pegasus/helpers/form_helpers.rb

```diff
@@ -93,6 +89,7 @@ def update_form(kind, secret, data)
 
   prev_data = JSON.parse(form[:data], symbolize_names: true)
   symbolized_data = Hash[data.map {|k, v| [k.to_sym, v]}]
+  symbolized_data.merge!(Hash[JSON.parse(symbolized_data[:data]).map {|k, v| [k.to_sym, v]}])
```

Comment:
This is necessary (bug-fix), else the new updates in the JSON `data` blob don't get pulled in to the update. The existing test confirming the update of `hoc_country_s` fails without this.